smithy-cli/src/main/java/software/amazon/smithy/cli/commands/Upgrade1to2Command.java

```diff
@@ -321,6 +323,16 @@ public Void stringShape(StringShape shape) {
             return null;
         }
 
+        @Override
+        public Void listShape(ListShape shape) {
+            getDefault(shape);
+            if (shape instanceof SetShape) {
+                writer.erase(shape.getSourceLocation(), 3); // `set` has 3 characters
+                writer.insert(shape.getSourceLocation(), "@uniqueItems" + System.lineSeparator() + "list");
+            }
+            return null;
+        }
```

Comment:
nit: overriding the `setShape()` visitor method will reduce the need for the `instanceof SetShape` check I think.

```suggestion
        @Override
        public Void setShape(SetShape shape) {
            getDefault(shape);
            writer.erase(shape.getSourceLocation(), 3); // `set` has 3 characters
            writer.insert(shape.getSourceLocation(), "@uniqueItems" + System.lineSeparator() + "list");
            return null;
        }
```